ruby/basics/arrays.md

```diff
@@ -34,28 +49,105 @@ Array.new(3, Array.new) #=> [[], [], []]
 ```
 
 ## Basic Methods
-Ruby gives you many methods to manipulate arrays and their contents, many of which are beyond the scope of this article. For full documentation, go to [http://ruby-doc.org/](http://ruby-doc.org/), click on "Core API", and scroll down to Classes: Array. There, you'll find the most up-to-date documentation on the various methods available Ruby arrays, along with explanations.
+Ruby gives you many methods to manipulate arrays and their contents, many of which are beyond the scope of this article. For full documentation, go to [http://ruby-doc.org/](http://ruby-doc.org/), click on "Core API", and scroll down to Classes: Array. There, you'll find the most up-to-date documentation on the various methods available to Ruby arrays, along with explanations.
 
-Calling `#methods` on an array will also yield a long list of methods, like so:
+Calling `#methods` on an array will also yield a long list of available methods, like so:
 
 ```
-x = [1, 2, 3, 4, 5]
-x.methods               #=> A very long list of methods
+num_array.methods       #=> A very long list of methods
+```
 
-* Accessing via index
-* Push/pop/shovel
-* Iteration: map/collect <!-- I'd leave enumerables to its own lesson-->
-* Addition/concatenation and substraction/difference
-* Other useful methods (empty?, include?, reverse, join)
+### Accessing element
+Like most other programming languages, Ruby arrays use **zero-based indexing**. As you learned in Chris Pine's tutorial, accessing an array's element is as simple as calling the `#[]` method with the **index**, or position, you desire. Recall that calling an invalid position will result in `nil`. Ruby also allows the use of negative indices, which return elements starting from the *end* of an array at [-1].
```

Comment:
yes, in ruby [] is a method, as are + - etc, but again these are kinda advanced concepts that we don't need to pollute the students with just yet. howabout "...an array's element is as simple as calling `myArray[x]` where `x` is the **index**, or position, you desire..."